php-frontend/src/main/java/org/sonar/php/tree/impl/SeparatedListImpl.java

```diff
@@ -84,7 +84,7 @@ public Object[] toArray() {
   }
 
   @Override
-  public <T> T[] toArray(T[] a) {
+  public <U> U[] toArray(U[] a) {
```

Comment:
You don't like the letter `T`? :joy:
`T` usually means `Type`, which seems to be a good fit here. Or am I missing another signification?

Reply:
Code Smell from sonar...(T was already used in outer scope)